code/modules/mob/living/carbon/give.dm

```diff
@@ -16,11 +14,11 @@
 		return
 
 	if(alert(target,"[usr] wants to give you \a [I]. Will you accept it?",,"Yes","No") == "No")
-		target.visible_message(span("warning", "\The [usr] tried to hand \the [I] to \the [target], \
-		but \the [target] didn't want it."))
+		target.visible_message("<b>[target]</b> pushes [usr]'s hand away.")
```

Comment:
might seem a bit aggressive, consider something like "x doesn't take y" or "x refuses the y"

Reply:
I mean, when you don't want something, you push the hand away. I don't see the aggressiveness here